proxy/logging/LogAccessHttp.cc

```diff
@@ -1623,14 +1623,15 @@ LogAccessHttp::marshal_http_header_field_escapify(LogField::Container container,
         fld = fld->m_next_dup;
 
         // Dups need to be comma separated.  So if there's another
-        // dup, then add a comma and a space ...
-        //
+        // dup, then add a comma and an escapified space ...
+        constexpr const char *SEP = ",%20";
```

Comment:
Should be char SEP[], the sizeof below will be wrong.

Reply:
Yeah, I flipped around several implementations. Oddly, I looked at the assembler output for this and it loaded the constant '4' for this. I'm not sure why, it looks like the compiler was smart enough to do what I meant. Perhaps because of the constexpr?